exo-go/src/app_dependency_helpers/exocom.go

```diff
@@ -1,12 +1,45 @@
-package appDependencyHelper
+package appDependencyHelpers
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
 	"github.com/Originate/exosphere/exo-go/src/types"
 )
 
 type exocomDependency struct {
 	config    types.Dependency
 	appConfig types.AppConfig
+	appDir    string
+}
+
+func (exocom exocomDependency) compileServiceRoutes() ([]map[string]interface{}, error) {
+	routes := []map[string]interface{}{}
+	serviceConfigs, err := serviceConfigHelpers.GetServiceConfigs(exocom.appDir, exocom.appConfig)
+	if err != nil {
+		return routes, err
+	}
+	serviceData := serviceConfigHelpers.GetServiceData(exocom.appConfig.Services)
+	for serviceName, serviceConfig := range serviceConfigs {
+		route := map[string]interface{}{
+			"role":     serviceName,
+			"receives": serviceConfig.ServiceMessages.Receives,
+			"sends":    serviceConfig.ServiceMessages.Sends,
+		}
+		namespace := serviceData[serviceName].NameSpace
+		if len(namespace) > 0 {
```

Comment:
Would result in the same thing but thoughts on using `if namespace != "" {`. I prefer that since `""` is the empty value for strings.